### PR TITLE
chore(expot validation): add translation functions to add and update forms

### PR DESCRIPTION
**Pull request by yusualhashash**
## How has this been tested?

- [ ] Prod
- [x] Development

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] Added all translations, no hard coded text. 
- [ ] Updated the UI (if needed).
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by ahmeterenodaci — APPROVED** [review]

**Review by eren-ayasofyazilim — APPROVED** [review]

File: apps/web/src/app/[lang]/app/[type]/operations/export-validation/new/form.tsx [hunk 1]
```diff
@@ -39,6 +40,36 @@ export default function Page({
...
+  const translatedForm = createFieldConfigWithResource({
+    schema:
+      $UniRefund_ExportValidationService_ExportValidations_CreateExportValidationDto,
+    resources: languageData,
+    extend: {
+      exportLocationId: {
+        renderer: (props) => (
+          <CustomCombobox<UniRefund_CRMService_Customss_CustomsProfileDto>
+            childrenProps={props}
+            emptyValue={languageData["Custom.Select"]}
```
Comment (ertugrulcan-ays) [inline apps/web/src/app/[lang]/app/[type]/operations/export-validation/new/form.tsx:52]: we can add translations for searchPlaceholder & searchResultLabel

File: apps/web/src/app/[lang]/app/[type]/operations/export-validation/[exportId]/form.tsx [hunk 2]
```diff
@@ -41,39 +42,46 @@ export default function Form({
...
+  const translatedForm = createFieldConfigWithResource({
+    schema:
+      $UniRefund_ExportValidationService_ExportValidations_UpdateExportValidationDto,
+    resources: languageData,
+    extend: {
+      exportLocationId: {
+        renderer: (props) => (
+          <CustomCombobox<UniRefund_CRMService_Customss_CustomsProfileDto>
+            childrenProps={props}
+            emptyValue={languageData["Custom.Select"]}
```
Comment (ertugrulcan-ays) [inline apps/web/src/app/[lang]/app/[type]/operations/export-validation/[exportId]/form.tsx:61]: we can add translations for searchPlaceholder & searchResultLabel

Comment (yusualhashash) [reply apps/web/src/app/[lang]/app/[type]/operations/export-validation/[exportId]/form.tsx:61]: done

File: apps/web/src/app/[lang]/app/[type]/operations/export-validation/new/form.tsx (hunk 1 above)
Comment (yusualhashash) [reply apps/web/src/app/[lang]/app/[type]/operations/export-validation/new/form.tsx:52]: done

**Review by a0m0rajab — APPROVED** [review]

**Review by ertugrulcan-ays — APPROVED** [review]

**Pull request merged**